packages/react-button/src/components/Button/useButtonStyles.ts

```diff
@@ -239,6 +239,12 @@ const useRootStyles = makeStyles({
       borderColor: 'transparent',
     },
   },
+  disabledFocusable: {
+    ['@media (forced-colors: active)']: {
```

Comment:
From doing a quick search, I see that safari does not support forced colours:
* https://caniuse.com/mdn-css_at-rules_media_forced-colors
* https://developer.mozilla.org/en-US/docs/Web/CSS/@media/forced-colors

Is there a difference way of handing HC On MacOS ?

Reply:
Yup, macOS doesn't have a forced-colors mode (hence the lack of support in Safari, since Safari doesn't exist on any OS that has it). Their high contrast works very differently, and doesn't forcibly override author-defined colors.